theano/gof/fg.py

```diff
@@ -416,6 +419,9 @@ def __import__(self, apply_node, check=True, reason=None):
             assert node not in self.apply_nodes
             self.__setup_node__(node)
             self.apply_nodes.add(node)
+            if not hasattr(node.tag, 'imported_by'):
+                node.tag.imported_by = []
+            node.tag.imported_by.append(str(reason))
```

Comment:
I know, this is to help debugging.

On Thu, May 12, 2016 at 12:06 PM, abergeron <notifications@github.com>
wrote:

> In theano/gof/fg.py
> <https://github.com/Theano/Theano/pull/4468#discussion_r63048106>:
>
> > @@ -416,6 +419,9 @@ def __import__(self, apply_node, check=True, reason=None):
> >              assert node not in self.apply_nodes
> >              self.__setup_node__(node)
> >              self.apply_nodes.add(node)
> > +            if not hasattr(node.tag, 'imported_by'):
> > +                node.tag.imported_by = []
> > +            node.tag.imported_by.append(str(reason))
>
> This doesn't seem like it's used anywhere
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/4468/files/8540c6d3e527f07faf9b5d791a80f7c029e3d71e#r63048106>
>